lib/api.js

```diff
@@ -13,10 +13,10 @@ class Api {
    * @param {string} options.version - Kubernetes API version
    * @param {string} options.namespace - Default namespace
    */
-  constructor(options) {
+  constructor(options, genericTypes) {
```

Comment:
Let's fold `genericTypes` into the `options` object to be consistent with the other arguments we're passing to this constructor (they're all passed via the `options` object)